### PR TITLE
[CDAP-15483] Adds back user information to navbar dropdown 

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-15483
Build: https://builds.cask.co/browse/CDAP-UDUT327

### CODE REVIEW

File: cdap-ui/app/cdap/components/AppHeader/AppToolBar/AppToolbarMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                        <IconSVG name="icon-user" />
+                        <a className={`${classes.usernameStyles} ${classes.linkStyles}`}>
+                          {this.state.username}
+                        </a>
+                      </MenuItem>
+                      <MenuItem
+                        onClick={this.toggleAccessTokenModal}
+                        className={classes.anchorMenuItem}
+                      >
+                        <a className={classes.linkStyles}> Access Token </a>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/AppHeader/AppToolBar/AppToolbarMenu.tsx]: i18n?

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Currently the bamboo build runs against latest develop which is why the builds were failing (with merge conflicts). Will fix them and will merge this PR.

**Comment by ajainarayanan** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-URUT160-5

**Pull request merged**